.github/workflows/verify-examples-sync.yml

```
@@ -0,0 +1,116 @@
+  discover:
+    name: Discover Examples
+    runs-on: ubuntu-latest
+    outputs:
+      examples: ${{ steps.list-examples.outputs.examples }}
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v4
+
+      - name: List example directories
+        id: list-examples
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2207:warning:3:14: Prefer mapfile or read -a to split command output (or quote to avoid splitting)

(shellcheck)